src/commands/Minion/cancel.ts

```diff
@@ -0,0 +1,68 @@
+import { KlasaMessage, CommandStore } from 'klasa';
+
+import { BotCommand } from '../../lib/BotCommand';
+import getActivityOfUser from '../../lib/util/getActivityOfUser';
+import removeSubTasksFromActivityTask from '../../lib/util/removeSubTasksFromActivityTask';
+import { Tasks, Activity, Time } from '../../lib/constants';
+import { requiresMinion } from '../../lib/minions/decorators';
+
+const options = {
+	max: 1,
+	time: 10000,
+	errors: ['time']
+};
+
+export default class extends BotCommand {
+	public constructor(store: CommandStore, file: string[], directory: string) {
+		super(store, file, directory, {
+			oneAtTime: true,
+			cooldown: 180
+		});
+	}
+
+	@requiresMinion
+	async run(msg: KlasaMessage) {
+		if (!msg.author.minionIsBusy) {
+			return msg.send(`${msg.author.minionStatus}`);
+		}
+
+		const currentTask = getActivityOfUser(this.client, msg.author);
+
+		if (!currentTask) {
+			throw `Your minion has no current task`;
+		}
+
```

Comment:
`${msg.author.minionName} isn't doing anything at the moment, so there's nothing to cancel.`